src/libs/environment/noMissingVariablesInEnvFiles.native.test.ts

```diff
@@ -46,4 +48,17 @@ describe('.env files', () => {
 
     expect(true).toBe(true) // Pass if no error is thrown
   })
+
+  test('.env file should match yup schema', () => {
+    for (const file of envFiles) {
+      const { parsed: env } = config({ path: file })
+      if (!env) {
+        throw new Error(`No variables found in ${file}`)
+      }
+
+      parseEnvironment(env)
+    }
+
+    expect(true).toBe(true) // Pass if no error
+  })
```

Comment:
Je suis pas très fan de ce test (surtout le `expect(true).toBe(true)`). Je te propose de faire comme ça (pour le premier test aussi)
```suggestion
  test('.env file should match yup schema', () => {
    const checkEnv = () => {
      for (const file of envFiles) {
        const { parsed: env } = config({ path: file })
        if (!env) {
          throw new Error(`No variables found in ${file}`)
        }

        parseEnvironment(env)
      }
    }

    expect(checkEnv).not.toThrow()
  })```